docs/knowledgebase/getting-started/glossary.md

```diff
@@ -43,59 +43,84 @@ overall amount of votes wins. Notably:
+[the Staking pallet](../runtime/frame#staking) from [FRAME](#frame)), authorities are determined
+through a token-weighted nomination/voting system.
 
-> We sometimes use "authorities" and "validators" to refer to what seems like the same thing.
+> The terms "authorities" and "[validators](#validator)" may sometimes seem to refer the same thing.
 > "Validators" is a broader term that can include other aspects of chain maintenance such as
 > parachain validation. In general authorities are a (non-strict) subset of validators and many
 > validators will be authorities.
 
 ## Aura (aka "Authority Round")
 
-Deterministic consensus protocol with Non-Instant Finality where block production is achieved by a
-rotating list of authorities that take turns issuing blocks over time, and where the majority of
-those online are honest authorities. See https://wiki.parity.io/Aura
+Deterministic [consensus](#consensus) protocol with non-instant [finality](#finality) where
+[block](#block) production is limited to a rotating list of [authorities](#authority) that take
+turns creating blocks; the majority of online authorities are assumed to be honest. Learn more by
+reading [the official wiki article](https://openethereum.github.io/wiki/Aura) for the Aura consensus
+algorithm.
 
 ### Aurand
 
-A variant of Aura where authorities are shuffled randomly on each round, increasing security.
+A variant of [Aura](#aura) where [authorities](#authority) are shuffled randomly on each round,
+increasing security.
 
 ### Aurand/Ouroboros
 
-Extension of Aurand where block production involves validators competing over limited slots that
-move very quickly, where most slots are not populated, but with rare slot collisions.
+Extension of [Aurand](#aurand) where block production involves [authorities](#authority) competing
+over limited slots that move very quickly; most slots are not populated, albeit with rare slot
+collisions.
 
-### Aurand+GRANDPA
+### Aurand + GRANDPA
 
-A hybrid consensus scheme where Aurand is used for block production and short-term
-probabilistic-finality, with long-term absolute finality provided through GRANDPA.
+A hybrid [consensus](#consensus) scheme where [Aurand](#aurand) is used for block production and
+short-term probabilistic-[finality](#finality), with long-term absolute finality provided through
+[GRANDPA](#grandpa).
 
 ---
 
 ## Blind Assignment of Blockchain Extension (BABE)
 
-Block authoring protocol similar to Aura where authorities are chosen with a verifiable random
-function (VRF) instead of a rotating list. Authorities are assigned a time slot in which they can
-select a chain and submit a new block for it. See
+[Block authoring](#author) protocol similar to [Aura](#aura) where [authorities](#authority) are
+chosen with a verifiable random function (VRF) instead of a rotating list. Authorities are assigned
+a time slot in which they can select a chain and submit a new block for it. See
```

Comment:
```suggestion
[Block authoring](#author) protocol similar to [Aura](#aura) where [authorities](#authority) win
time slots based on a verifiable random function (VRF) instead of a rotating list. The winning authority
can select a chain and submit a new block for it. See
```